src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py

```diff
@@ -579,7 +579,7 @@ def load_acs_service_principals(config_path):
         return None
 
 
-# pylint: disable-msg=too-many-arguments
+# pylint: disable-msg=too-many-statements
```

Comment:
I do not recommend this. There is a reason for pylint rule of __too many statements__ to facilitate readable coding style. Can you break this method into smaller pieces?

Reply:
I am inclined to do it in a different PR, rather than this one. :)

Reply:
Ok, then. I'll keep an eye on this piece of code then :)